### DISCUSSION TITLE
How to reorder the tweets retrieved by api.favorites

### DISCUSSION THREAD

**Discussion by NTHieu95** [tweepy/tweepy#1397] [Questions] [answered]
I'm using this method to retrieve my favorite tweets:
`for favorite in tweepy.Cursor(api.favorites, id=User, tweet_mode="extended").items(3200):`

But then I realized it was ordered by the date of the newest tweets to the oldest tweets, not from my lastest favorite tweets to my oldest favorite tweets (which is the way I want it).
What I'd like to ask is: Is there any way to reorder the tweets retrieved by the API? Thanks a lot in advance.

**Comment by Harmon758** [comment, marked as answer]
Unfortunately, that's not something that Twitter's API supports.
See https://twittercommunity.com/t/specify-sort-order-of-favorites-from-rest-api/118776 and https://twittercommunity.com/t/favorites-list-doesnt-add-newly-favorited-tweets-to-the-top/31206.
